### PR TITLE
gprof: add page

### PR DESCRIPTION
**Pull request by RaphGL**
Added basic instructions for how to use the unix gprof C profiler

### CODE REVIEW

File: pages/common/gprof.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# gprof
+
+> Performance analysis tool for many programming languages
```
Comment (kbdharun) [inline pages/common/gprof.md:3]:
```suggestion
> Performance analysis tool for many programming languages.
```

File: pages/common/gprof.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# gprof
+
+> Performance analysis tool for many programming languages  
+> It profiles the function executions of a program
```
Comment (kbdharun) [inline pages/common/gprof.md:4]:
```suggestion
> It profiles the function executions of a program.
```

File: pages/common/gprof.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# gprof
+
+> Performance analysis tool for many programming languages.
+> It profiles the function executions of a program.
+
+- Compile binary with gprof information and run it to get {{gmon.out}}:
```
Comment (pixelcmtd) [inline pages/common/gprof.md:6]:
```suggestion
- Compile binary with gprof information and run it to get `gmon.out`:
```

**Review by pixelcmtd — CHANGES_REQUESTED** [review]
Please include a "More information" link.

**Comment by RaphGL** [comment]
I've added a link for more information

File: pages/common/gprof.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# gprof
+
+> Performance analysis tool for many programming languages.
+> It profiles the function executions of a program. 
+> More information: https://ftp.gnu.org/old-gnu/Manuals/gprof-2.9.1/html_mono/gprof.html
```
Comment (pixelcmtd) [inline pages/common/gprof.md:5]:
```suggestion
> More information: <https://ftp.gnu.org/old-gnu/Manuals/gprof-2.9.1/html_mono/gprof.html>
```

File: pages/common/gprof.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# gprof
+
+> Performance analysis tool for many programming languages.
+> It profiles the function executions of a program.
```
Comment (kbdharun) [inline pages/common/gprof.md:4]:
```suggestion
> It profiles the function executions of a program.
```

**Comment by RaphGL** [comment]
Everything is in order now. Awaiting your approval.

**Review by pixelcmtd — APPROVED** [review]

File: pages/common/gprof.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+- Compile binary with gprof information and run it to get `gmon.out`:
+
+`gcc -pg {{program.c}} && {{./a.out}}`
+
+- Run gprof to obtain profile output:
+
+`gprof`
+
+- Suppress profile fields' description:
```
Comment (kbdharun) [inline pages/common/gprof.md:15]:
```suggestion
- Suppress profile field's description:
```
Is this better?

**Review by kbdharun — APPROVED** [review]

**Pull request merged**